app/controllers/samples_controller.rb

```diff
@@ -87,8 +83,6 @@ def edit # rubocop:todo Metrics/AbcSize
     end
   end
 
-  # rubocop:enable Metrics/MethodLength
-
   # rubocop:todo Metrics/MethodLength
```

Comment:
Can that comment go too?
Also, does the methodlength change now factor into AbcSize calculation so some of those can be removed?